python/src/gluon_ir.cc

```diff
@@ -762,7 +762,18 @@ void init_gluon_ir(py::module &&m) {
               tt::CacheModifier cacheModifier) {
              self.create<ttag::BufferLoadToLocalOp>(
                  dest, ptr, offsets, mask, other, stride, cacheModifier);
-           });
+           })
+      .def("create_async_tdm_copy_global_to_local",
+           [](GluonOpBuilder &self, Value descPtr, std::vector<Value> &indices,
+              Value result) {
+             Value pred = self.create<arith::ConstantIntOp>(1, 1);
+             self.create<ttag::AsyncTDMCopyGlobalToLocalOp>(descPtr, indices,
+                                                            result, pred);
+           })
+      .def("create_async_tdm_store_wait", [](GluonOpBuilder &self, int num) {
```

Comment:
Should we just name it as `create_async_tdm_wait` to be consistent with the op name?